src/components/common/MultiSelect.tsx

```diff
@@ -9,57 +9,57 @@ interface styledProps {
 }
 
 const S = styled(Select)<styledProps>`
-background:#ffffff00;
-border: 1px solid ${(p: any) => p.color && p.color.grayish.G750};
-color: ${(p: any) => p.color && p.color.pureBlack};
-box-sizing: border-box;
-box-shadow:none;
-border: none !important;
-user-select:none;
-font-size:12px;
-border-width:0px !important;
+  background: #ffffff00;
+  border: 1px solid ${(p: any) => p.color && p.color.grayish.G750};
+  color: ${(p: any) => p.color && p.color.pureBlack};
+  box-sizing: border-box;
+  box-shadow: none;
+  border: none !important;
+  user-select: none;
+  font-size: 12px;
+  border-width: 0px !important;
+  z-index: 909;
```

Comment:
this is a brittle solution what happens if we have another modal or screen that needs to be on top?